packages/circuit-ui/components/NotificationModal/NotificationModal.tsx

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright 2019, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { css, ClassNames } from '@emotion/react';
+import { ReactNode } from 'react';
+import ReactModal from 'react-modal';
```

Comment:
I might have missed this discussion, but was it too complicated to reuse our ReactModal implementation from the Modal component here? I assume there's a good reason we're duplicating some of the code

Reply:
They look different on mobile. The Modal transforms into a bottom sheet, the NotificationModal doesn't.

Reply:
Did GC explain why? Fine by me but it seems inconsistent

Reply:
I will add this as a note, so we can check with GC.